hawp/detector.py

```diff
@@ -120,14 +120,18 @@ def forward(self, images, annotations = None):
         #jloc_pred_nms = non_maximum_suppression(jloc_pred[0])
         #topK = torch.clamp((jloc_pred_nms > 0.008).count_nonzero(), max=300)
 
-        juncs_pred, _ = get_junctions(non_maximum_suppression(jloc_pred[0]),joff_pred[0], topk=300, th=0.008)
+        nms_jloc_pred = non_maximum_suppression(jloc_pred)[0]
```

Comment:
This is the part where I pass the 4 dimensional tensor, and then take out the batch dim after max_pool2d